daras_ai_v2/asr.py

```diff
@@ -1027,11 +1075,7 @@ def audio_url_to_wav(audio_url: str) -> tuple[str, int]:
         raise_for_status(r, is_user_url=True)
 
     wavdata, size = audio_bytes_to_wav(r.content)
-    if not wavdata:
```

Comment:
Need to handle this case

Reply:
given how complex this is getting, it might be easier to ship the inefficient version that uploaded and downloaded the file for ghana nlp